#!/usr/bin/python3

def raise_exception_msg(message=""):
    """Raises the NameError exception with a message"""
    raise NameError(message)
